_includes/meta.html

```diff
@@ -16,15 +16,28 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <!-- Title and meta description
     ================================================== -->
-  <title>{{site.title}} - {{ page.title}}</title>
-  <meta property="og:title" content="{{site.title}}">
-  <meta name="description" content="{{site.description}}">
-  <meta property="og:description" content="{{site.description}}">
+  {% capture title %}
+    {%- if page.title -%}{{- page.title | strip_html }} | {% endif -%}{{ site.title -}}
```

Comment:
Are you interested in inserting "Pages" for the part of the site that's specific to Pages?